test/e2e/cache/cache_test.go

```diff
@@ -272,6 +272,8 @@ func TestCacheWithServerLifecycle(t *testing.T) {
 	testifyAssert.True(t, kubernetes.IsOwnedBy(cr, ispn), "Cache has unexpected owner reference")
 
 	// Update cache configuration via REST
+	// Await all creation related updates are gone
+	time.Sleep(10 * time.Second)
```

Comment:
Instead of always waiting 10 seconds we could retry `cacheHelper.CacheClient.UpdateConfig(..)` until it succeeds or times out after 10 seconds.

Reply:
> Instead of always waiting 10 seconds we could retry `cacheHelper.CacheClient.UpdateConfig(..)` until it succeeds or times out after 10 seconds.

you mean an ad hoc WaitForTemplateUpdate with the above policy?

Reply:
Yeah exactly